heroku-24/setup.sh

```diff
@@ -43,6 +43,9 @@ packages=(
   bzip2
   curl
   file
+  fonts-dejavu-core
+  fonts-dejavu-mono
```

Comment:
Please could you add comments to these? In general we should have comments for all non-obvious packages in this list.

Also, the `fonts-dejavu-mono` package is a transitive dependency of `fonts-dejavu-core`, and we now intentionally avoid listing transitive dependencies here - so could you remove `fonts-dejavu-mono` from this list?